### DISCUSSION TITLE
Multi-User Stellarium

### DISCUSSION THREAD

**Discussion by Benach** [Stellarium/stellarium#3190] [Ideas]
At my astronomy club we use Stellarium as the default planetarium program for our domes (Yes, plural). We have many users for each dome. Wouldn't it be an idea to create a option to create a multi-user add-on that allows every user to open, edit, save their own preferred settings? It would of course also require a maintainer/superuser account. But the advantage is that this makes sure everybody has their own preferred settings.

**Comment by gzotti** [comment]
Settings are stored per account. If you use separate user accounts, you will have separate settings. If all users share one account, you can use the --user-dir and --config-file options.

**Comment by Benach** [comment]
Thanks. Will check that out later next week.
